src/hotspot/share/gc/z/zStat.hpp

```diff
@@ -546,6 +546,7 @@ class ZStatHeap : public AllStatic {
   static size_t max_capacity();
   static size_t used_at_mark_start();
   static size_t used_at_relocate_end();
+  static size_t live();
```

Comment:
Please call this `live_at_mark_end()` to match the names of the neighboring functions.

Reply:
👍